frame/assistants/pangoro-subxt/scripts/codegen.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
Remove this script/dirctory and use `scripts/subxt-gen.sh` script, if that script file does not have to execute permission please set it.